src/rgw/rgw_op.cc

```diff
@@ -366,8 +366,9 @@ WRITE_CLASS_ENCODER(multipart_upload_info)
 
 static int get_multipart_info(struct req_state *s,
 			      rgw::sal::RGWObject* obj,
-                              multipart_upload_info *upload_info)
+                              multipart_upload_info *upload_info, const jaeger_tracing::Span* const parent_span = nullptr)
```

Comment:
you have the span in req_state, why do you also need it as a parameter?

Reply:
the span in req_state is just a root_span, which wont serve the purpose of parent_span here